search_service/api/document.py

```diff
@@ -0,0 +1,76 @@
+import logging
+
+from typing import Tuple, Any
+
+from flask_restful import Resource, reqparse
+from search_service.proxy import get_proxy_client
+from search_service.models.table import TableSchema
+from search_service.api.table import TABLE_INDEX
+
+LOGGER = logging.getLogger(__name__)
+
+
+class DocumentAPI(Resource):
+
+    def __init__(self) -> None:
+        self.proxy = get_proxy_client()
+        self.parser = reqparse.RequestParser(bundle_errors=True)
+        self.parser.add_argument('index', required=False, default=TABLE_INDEX, type=str)
+        super(DocumentAPI, self).__init__()
+
+    def post(self) -> Tuple[Any, int]:
+        """
+        Uses the Elasticsearch bulk API to load data from JSON. Uses Elasticsearch
+        index actions to create or update documents by id
+
+        :param data: list of data objects to be indexed in Elasticsearch
+        :return: name of new index
+        """
+        self.parser.add_argument('data', required=True)
+        args = self.parser.parse_args()
+
+        try:
+            data = TableSchema(many=True, unknown='EXCLUDE').loads(args.get('data'))
+            results = self.proxy.create_document(data=data, index=args.get('index'))
+            return results, 200
```

Comment:
nit: Hi, how about  `return results, HTTPStatus.OK` with `from http import HTTPStatus`

Reply:
sure, I updated it!